pages/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Head from 'next/head'
+import styles from '../styles/Home.module.css'
+
+export default function HomePage() {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>GovsStack testing</title>
```

Comment:
Do we want to use translations? Or add them later?

Reply:
I will ad them later